object.go

```diff
@@ -756,7 +756,7 @@ func queryObjects(c context.Context, opts *ObjectQueryOptions) (Iterator, error)
 		cursors = append(cursors, cursor)
 	}
 
-	trace.FromContext(c).SetLabel("wp/object/count", strconv.Itoa(len(ids)))
+	trace.FromContext(c).AddAttributes(trace.Int64Attribute("wp/object/count", int64(len(ids))))
```

Comment:
Please be consistent with the other instances where `strconv.Itoa` was used. Either change them all to `trace.Int64Attribute`s or leave them as `trace.StringAttribute`s.

I think, however, `trace.Int64Attribute` would be more preferrable.

Reply:
My intention was to use `trace.Int64Attribute`. I thought I ran the regex replace everywhere. Will do so now.